### PR TITLE
Handling cell details deep link

### PR DESCRIPTION
**Pull request by pantelisss**
## **Why?**
Handle in app universal link with `/cells` path
### **How?**
- Added the case in `DeepLinksHandler`
- Resolve cell center in for a cell in `DeepLinksHandler`
### **Testing**
- Run the `release` scheme and open an explorer link
### **Additional context**
fixes fe-707

### CODE REVIEW

File: wxm-ios/DomainLayer/DomainLayer/UseCases/ExplorerUseCase.swift [hunk 1]
```diff
@@ -53,6 +53,21 @@ public class ExplorerUseCase {
...
 	}
 
+	public func getCell(cellIndex: String) async throws -> Result<PublicHex?, NetworkErrorResponse> {
+		let publisher = try explorerRepository.getPublicHexes()
+		return await withUnsafeContinuation { continuation in
+			publisher.sink { response in
+				switch response.result {
+					case .success(let cells):
+						let cell = cells.first(where: { $0.index == cellIndex })
+						continuation.resume(returning: .success(cell))
```
Comment (PavlosTze) [inline wxm-ios/DomainLayer/DomainLayer/UseCases/ExplorerUseCase.swift:63]: What will happen if the cell is null (no matching index is found) ? I think we should handle it with the respective error message as it's a failure.

**Comment by PavlosTze** [comment]
Also clicking back from the cell station details doesn't open the explorer (and navigate to the respective cell), is that OK behavior?

File: wxm-ios/DomainLayer/DomainLayer/UseCases/ExplorerUseCase.swift (hunk 1 above)
Comment (pantelisss) [reply wxm-ios/DomainLayer/DomainLayer/UseCases/ExplorerUseCase.swift:63]: Do we have a message for this?

Comment (PavlosTze) [reply wxm-ios/DomainLayer/DomainLayer/UseCases/ExplorerUseCase.swift:63]: I used this:
`Could not find the requested explorer cell`

**Comment by PavlosTze** [comment]
> > Also clicking back from the cell station details doesn't open the explorer (and navigate to the respective cell), is that OK behavior or should it work otherwise?
> 
> It's the same approach with station details. I think UX-wise is the expected. Also, a functionality changing tabs and pushing a screen becomes more error-prone

I see, I have this functionality in both station & cells, but I think when we had discussed it back then it was just some kind of bonus if it's easily implemented. So I think we are good with proceeding as is, let's just implement only the other one above, regarding a cell not found.

File: wxm-ios/DomainLayer/DomainLayer/UseCases/ExplorerUseCase.swift (hunk 1 above)
Comment (pantelisss) [reply wxm-ios/DomainLayer/DomainLayer/UseCases/ExplorerUseCase.swift:63]: Done

**Review by PavlosTze — APPROVED** [review]

**Pull request merged**